pkg/controller/srcdst_controller_test.go

```diff
@@ -48,14 +49,18 @@ func TestDisableSrcDstIfEnabled(t *testing.T) {
 	ec2Client := NewMockEC2Client()
 	kubeClient := fake.NewSimpleClientset(&v1.NodeList{Items: []v1.Node{*node0, *node1}})
 
+	var my_opts = common.K8sEc2SrcdstOpts{
```

Comment:
It also seems like we should add a test for this new functionality when `PatchNode = true`